file_formats/ie_formats/itm_v1.htm

```diff
@@ -1747,12 +1747,12 @@
         </tr>
         <tr>
           <td>2</td>
-          <td> <span class="unknown">Unknown</span></td>
+          <td> Damage strength bonus (BGEE)</td>
```

Comment:
Are these two different from "Add strength bonus" at bit 0 that they just split the bonus? If you want to apply just the to-hit or the damage one, while 0 does both?

Reply:
> Are these two different from "Add strength bonus" at bit 0 that they just split the bonus? If you want to apply just the to-hit or the damage one, while 0 does both?

Yes, they allow for splitting the bonus (EE only though)

Reply:
Please add that to their descriptions then.

Reply:
Also, the original ignored the strength to-hit bonus for ranged weapons for that bit. Is that still true with the split bits or do they override this behaviour?